unitylibs/core/workflow/workflow-upload/action-binder.js

```diff
@@ -53,6 +55,7 @@ class ServiceHandler {
       alertText.innerText = msg;
       errorToast.classList.add('show');
     });
+    window.lana?.log(`Message: ${msg}, Error: ${errorUpdate}`, lanaOptions);
```

Comment:
nit: we could remove the `const errorUpdate ` above and just do:
```suggestion
    window.lana?.log(`Message: ${msg}, Error: ${error || ''}`, lanaOptions);
```